docs/src/pages/Conformance.md

```diff
@@ -124,26 +123,52 @@ The whole system history interaction can be used to obtain a list of changes (cr
     curl -k -u '<username>:<password>' 'https://<host>:<port>/fhir-server/api/v4/_history'
 ```
 
-The response is a history bundle as described by the [FHIR specification](https://www.hl7.org/fhir/http.html#history) with one exception. Whereas the specification requires each entry in the history bundle to contain the full contents of the resource (at least for entries with a `entry.request.method` of PUT or POST), the IBM FHIR Server whole-system history response bundle contains only references to the resources. Clients may choose which resources they fetch. They may also fetch the resources in parallel, which may greatly improve throughput.
+The response is a history bundle as described by the [FHIR specification](https://www.hl7.org/fhir/http.html#history) with one exception. Whereas the specification requires each entry in the history bundle to contain the full contents of the resource (at least for entries with a `entry.request.method` of PUT or POST), clients can specify the HTTP header value `Prefer: return=none` in which case the whole-system history response bundle contains only references to the resources. This allows clients to decide which resources they actually need to read.
 
-To return all changes that have occurred since a known point in time, use the `_since` query parameter:
+There are three traversal paths through the data which are requested using the `_sort` request parameter:
+
+1. `_sort=-_lastUpdated` [Default] Decreasing time - most recent changes first
+2. `_sort=_lastUpdated` Increasing time - oldest changes first
+3. `_sort=none` Increasing id order - oldest changes first
+
+Option 1 is useful for users interested in finding the most recent changes that have been applied to the server. Option 2 is useful for system-to-system synchronization because it represents the change history of the server. Option 3 is also useful for system-to-system synchronization cases, and benefits guaranteeing uniqueness in the resources being returned when following the `next` links.
```

Comment:
minor wording suggestion 
```suggestion
Option 1 is useful for users interested in finding the most recent changes that have been applied to the server. Option 2 is useful for system-to-system synchronization because it represents the change history of the server. Option 3 is also useful for system-to-system synchronization cases guarantees uniqueness across pages when following the `next` links (as opposed to option 2 which may repeat entries at the time window boundary).
```

Reply:
good